cmd/controller/example/README.md

```diff
@@ -1,6 +1,16 @@
 # Installing the skupper controller
 
 ```
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_certificate_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_claim_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_cluster_policy_crd.yaml
```

Comment:
Actually we don't want the policy CRD. It is from v1. The current CRDs need to be separated at some point, with the v2 ones pulled into their own grouping. For now, we can just not install the policy crd in this guide.

Reply:
I’ve removed that CRD from the list.